static/js/components/CommentForms.js

```diff
@@ -158,17 +159,33 @@ const mapDispatchToProps = (dispatch, ownProps) => {
     beginEditing: beginEditing(dispatch, formKey),
     onSubmit:     R.curry((postID, text, commentID, post, e) => {
       e.preventDefault()
-      return dispatch(
-        actions.comments.post(postID, text, commentID)
-      ).then(() => {
-        dispatch(
-          setPostData({
-            ...post,
-            num_comments: post.num_comments + 1
-          })
-        )
-        cancelReply(dispatch, formKey)()
-      })
+      return dispatch(actions.comments.post(postID, text, commentID))
+        .then(() => {
+          dispatch(
+            setPostData({
+              ...post,
+              num_comments: post.num_comments + 1
+            })
+          )
+          cancelReply(dispatch, formKey)()
+        })
+        .catch(err => {
+          if (err.errorStatusCode === 410) {
+            // Comment was deleted
+            dispatch(
+              setSnackbarMessage({
+                message: "Comment you are replying to was deleted"
```

Comment:
Might suggest 'This comment has been deleted and cannot be replied to' or something similar, although @roberthouse54 might have a better idea of what makes sense than me :smiley: